tests/system/providers/sftp/example_sftp_sensor.py

```diff
@@ -76,8 +76,18 @@ def sftp_sensor_decorator():
     sftp_with_operator = SFTPSensor(task_id="sftp_operator", path=FULL_FILE_PATH, poke_interval=10)
     # [END howto_operator_sftp_sensor]
 
+    # [START howto_sensor_sftp_async]
```

Comment:
as we're not using the term `async` in the sensor, I might suggest we change it to deferrable.
```suggestion
    # [START howto_sensor_sftp_deferrable]
```

Reply:
Make sense, made the required change here - https://github.com/apache/airflow/pull/37117/commits/1b162e76875e4de1a4083e9af7e5bdde5b564899